src/components/navigation/www/navigation.jsx

```diff
@@ -225,99 +225,101 @@ var Navigation = React.createClass({
                             <Input type="hidden" name="sort_by" value="datetime_shared" />
                         </form>
                     </li>
-                    {this.props.session.user ? [
-                        <li className="link right messages" key="messages">
-                            <a
-                                href="/messages/"
-                                title={formatMessage({id: 'general.messages'})}>
+                    {this.props.session.status == sessionActions.Status.FETCHED ? (
```

Comment:
Use `===` for comparison — it gives more predictable results when the values you're comparing are different types because it doesn't do implicit type conversion.  For example,

```
'' == '0'           // false
0 == ''             // true
0 == '0'            // true

false == 'false'    // false
false == '0'        // true

false == undefined  // false
false == null       // false
null == undefined   // true

' \t\r\n ' == 0     // true
```